articles/communication-services/quickstarts/voice-video-calling/includes/raw-media/raw-media-access-ios.md

```diff
@@ -374,7 +374,7 @@ Because the app generates the video frames, the app must inform the Azure Commun
     ```swift
     var videoStreamFormat = VideoStreamFormat()
     videoStreamFormat.resolution = VideoStreamResolution.p360
-    videoStreamFormat.pixelFormat = VideoStreamPixelFormat.rgba
+    videoStreamFormat.pixelFormat = VideoStreamPixelFormat.nv12
```

Comment:
Why do we need to update this to NV12? 
If we need too, we also need to change the stride line bellow.